flow-typed/redux.js

```diff
@@ -0,0 +1,74 @@
+declare module 'redux' {
+
+  /*
+
+    S = State
+    A = Action
+
+  */
+
+  declare type Dispatch<S, A> = (action: A) => A;
```

Comment:
Good point. Is there a better option than returning `any`?

Note. The typescript definition file should be modified as well